jwcrypto/jwk.py

```diff
@@ -387,6 +387,19 @@ def import_key(self, **kwargs):
                                               ' "key_ops" values specified at'
                                               ' the same time')
 
+    @classmethod
+    def from_json(cls, key):
+        """Creates a RFC 7517 JWK from the standard JSON format.
+
+        :param key: The RFC 7517 representation of a JWK.
+        """
+        obj = cls()
+        try:
+            jkey = json_decode(key)
+        except Exception:  # pylint: disable=broad-except
+            raise InvalidJWKValue()
```

Comment:
Python 2 doesn't contain the exception context. Could you please include the inner exception inside the outer exception (``except Exception as e: raise InvalidJWKValue(e)``.

Reply:
Good catch